src/test/java/org/candlepin/model/ConsumerTest.java

```diff
@@ -692,6 +692,17 @@ public void testGetOwnerKeyWithNoOwner() {
         assertNull(consumer.getOwnerKey());
     }
 
+    @Test
+    public void testConsumerCloudData() {
```

Comment:
This test name should be `testSetConsumerCloudData`

Reply:
done